packages/features/calendars/DestinationCalendarSelector.tsx

```diff
@@ -115,6 +115,11 @@ const DestinationCalendarSelector = ({
           value: `${cal.integration}:${cal.externalId}`,
         })),
     })) ?? [];
+
+  // Get primary calendar, which is shown in the placeholder since this is the calendar that will
+  // be used when no destination calendar is selected.
+  const primaryCalendar = query.data.connectedCalendars.filter((cal) => Boolean(cal.primary))[0];
```

Comment:
If there is NO calendar connected, there's not destinationCalendar in the `eventType` object. We rather need to get the default from the calendars in that case. That is what this change does.